Documentation/ceph-cluster-crd.md

```diff
@@ -24,6 +24,13 @@ spec:
     # see the "Cluster Settings" section below for more details on which image of ceph to run
     image: ceph/ceph:v13.2.2-20181023
   dataDirHostPath: /var/lib/rook
+  # set the amount of mons to be started
```

Comment:
To keep the example yaml simple I would suggest we don't add comments unless they are absolutely necessary. The comment on the ceph image is rather critical to make sure users find the version they need, but I think the number of mons and enabling the dashboard are self explanatory.